package.json

```diff
@@ -20,25 +20,56 @@
     "url": "https://github.com/mjbvz/vscode-github-markdown-preview-style/issues"
   },
   "engines": {
-    "vscode": "^1.15.0"
+    "vscode": "^1.41.0"
   },
   "categories": [
     "Other"
   ],
+  "activationEvents": [
+    "onLanguage:markdown",
+    "onWebviewPanel:markdown.preview",
+    "onCustomEditor:vscode.markdown.preview.editor"
+  ],
+  "main": "dist/extension.js",
+  "browser": "dist/extension.js",
   "contributes": {
+    "markdown.markdownItPlugins": true,
     "markdown.previewStyles": [
-      "./base.css",
-      "./github-markdown.css",
-      "./node_modules/highlight.js/styles/github.css"
+      "./dist/base.css",
+      "./dist/github-markdown.css",
+      "./dist/highlight-js.css"
+    ],
+    "configuration": [
+      {
+        "title": "Markdown Preview Github Styling",
+        "properties": {
+          "markdown-preview-github-styles.colorMode": {
+            "type": "string",
+            "description": "Color mode for the styling of the Markdown preview.",
+            "enum": [
+              "auto",
+              "system",
+              "light",
+              "dark"
+            ],
+            "enumDescriptions": [
+              "Automatically match the editor color theme.",
+              "Sync to the system/OS color theme.",
+              "Always use the Github site's light theme.",
+              "Always use the Github site's dark theme."
+            ],
+            "default": "auto"
+          }
+        }
+      }
     ]
   },
-  "dependencies": {
-    "highlight.js": "^10.7.2"
-  },
   "devDependencies": {
-    "github-markdown-css": "^4.0.0"
+    "eslint": "^7.20.0",
```

Comment:
Remove eslint since it is not related to this change. Would be to see it added in another PR though

Reply:
Done as of b2c8012